rust/build.rs

```diff
@@ -1,6 +1,10 @@
 // Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Download the protoc and set the path for tonic_build.
+    let protoc_path = protoc_bin_vendored::protoc_bin_path().unwrap();
+    std::env::set_var("PROTOC", protoc_path.as_os_str());
```

Comment:
If `protoc` actually exists in local? should we avoid downloading?
Maybe we should check it at first, for example:
```
 if let Err(_) = which::which("protoc") {
      download
}
```

Reply:
I guess no need because it is possible that local protoc can't meet the requirement for the version.